nDPI-patch/README.md

```diff
@@ -212,16 +212,16 @@ make[2]: Leaving directory '/usr/src/linux-headers-5.8.0-1040-generic'
 giving you insight on which kernel version you are building with.
 
 To rollback:
-1. Install 5.4 kernel then reboot
+1. Install the 5.4 kernel then reboot
     ```
     $ sudo apt install linux-generic
     $ reboot
     ```
-1. Log back in and list the kernel packages
+1. Log back in and list the kernel packages for the version you are using
```

Comment:
"are" should be "were" here.  Or am I reading it wrong?

Reply:
I think it's 'are' since the rollback isn't complete until you remove the packages. Maybe it needs to be reworded to be more clear.

"... for the version you are moving from"

Reply:
Updated.